doc/sphinx/dev-guide/index.rst

```diff
@@ -6,21 +6,6 @@ The Varnish Developers Guide
 This is the deliberately short and to the point list of things
 Varnish Developers should know.
 
-Behaviour
----------
-
-* Be sensible.
-
-* If in doubt, think.
-
-* If still in doubt, ask.
-
-* Admit your mistakes, it's faster that way.
-
-* Thou SHALL not paint `bikesheds. <http://bikeshed.org/>`_
-
-* We will toss you out of the project rather than add another rule.
-
 Technical stuff
```

Comment:
This cannot be removed until it is replaced with something else.

The last line in particular is important, both now and in whatever replaces this:  Trying to reign in sociopaths by circumscribing their actions post-factum is not a workable model.